Lib/calendar.py

```diff
@@ -723,7 +723,7 @@ def main(args):
     parser.add_argument(
         "-L", "--locale",
         default=None,
-        help="locale to be used from month and weekday names"
+        help="locale to use for month and weekday names"
```

Comment:
Thomas may want this reverted as it is a 'non-docs' change

Reply:
Thanks for the consideration, but I'll consider help output to be docs :)